app/gui/src/presenter/graph.rs

```diff
@@ -147,6 +146,26 @@ impl Model {
         self.state.update_from_view().set_node_expression(id, expression);
     }
 
+    /// Part of node expression was edited in the view. Should be called whenever the user partially
+    /// changes the contents of a node during editing, e.g. via a widget.
+    fn node_port_expression_set(
```

Comment:
Why `node_port_expression_set` function doc state that "Part of node expression was edited in the view"? How this function knows that this happened? Also, what does it mean that "Should be called whenever the user partially changes the contents ..." - should it always be called then? 

Anyway, docs should semantically describe what a function does, not describe its possible use-side contexts. These docs do not describe what this function does.